proposals/wac-ucr/index.bs

```diff
@@ -76,7 +80,23 @@ individuals (i.e. Alice, Bob, Carol, etc.) are already
 [=authenticated agents=], and their corresponding [=identities=] are known
```

Comment:
The full text here is 
"For the purposes of simplicity, the use cases herein assume that named individuals (i.e. Alice, Bob, Carol, etc.) are already authenticated"
Not all of them do, so I made a PR to make a small fix for this: https://github.com/solid/authorization-panel/pull/166/commits/01434e6cc8a644422a165e6aa6c31003a47a2504

Reply:
[That](https://github.com/solid/authorization-panel/commit/01434e6cc8a644422a165e6aa6c31003a47a2504)'s not a PR, it's a commit to (I think) your fork of this branch... (Click https://github.com/solid/authorization-panel/commit/01434e6cc8a644422a165e6aa6c31003a47a2504 to see what github says...)

Reply:
Ah yes, I was trying to work out how the `gh` command line interface works, so that I can work from the command line. It is difficult not to work from the cli because one needs to run `bikeshed` to be able to really understand the context of the changes.

Reply:
But we covered that in todays meeting. See the notes here: https://github.com/solid/authorization-panel/pull/167/files?short_path=e66d5b0#diff-e66d5b08e777fcb77d751f43b35a40f779e28776d19fde98601f334c1105e56a